### PR TITLE
skip multi-gpu test when running on single-gpu machine

### PR DESCRIPTION
**Pull request by awaelchli**
a test that requires 2 gpus should be skipped when running on single-gpu system.

### CODE REVIEW

**Review by Borda — APPROVED** [review]

File: tests/models/test_gpu.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def test_multi_gpu_none_backend(tmpdir):
     tpipes.run_model_test(trainer_options, model)
 
 
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires 2 GPUs")
```
Comment (rohitgr7) [inline tests/models/test_gpu.py:52]:
```suggestion
@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
```
just putting the same `reason` as it's there in other tests.

**Review by rohitgr7 — APPROVED** [review]

File: tests/models/test_gpu.py (hunk 1 above)
Comment (awaelchli) [reply tests/models/test_gpu.py]: awesome

File: tests/models/test_gpu.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def test_multi_gpu_none_backend(tmpdir):
     tpipes.run_model_test(trainer_options, model)
 
 
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
 @pytest.mark.parametrize('gpus', [1, [0], [1]])
 def test_single_gpu_model(tmpdir, gpus):
```
Comment (SkafteNicki) [inline tests/models/test_gpu.py:54]: Is the test description wrong then? It say "test_single_gpu"

Comment (awaelchli) [reply tests/models/test_gpu.py:54]: it's a test running on a single gpu, but on gpu index "1", therefore this requires a machine with at least 2 gpus.

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**